querybook/server/datasources/ai_assistant.py

```diff
@@ -22,3 +22,18 @@ def query_auto_fix(query_execution_id):
     )
 
     return Response(res_stream, mimetype="text/event-stream")
+
+
+@register("/ai/generate_query/", custom_response=True)
+def generate_sql_query(
+    query_engine_id: int, tables: list[str], question: str, data_cell_id: int = None
```

Comment:
instead of data cell, use original_query so it is not limited to data doc?

Reply:
The problem is I realized that for GET http request, it has length limit on the query string. So it may not be safe to pass along the whole query.

Can reconsider the EventSource solution in future and change it back to use POST method. For this specific case, I can move the get query from the data cell into this api endpoint and make `ai_assistant` to accept the original query.